stage1/rootfs/usr_from_kvm/kernel/patches/reboot.patch

```diff
@@ -0,0 +1,22 @@
+--- linux-4.0/kernel/reboot.c~	2015-04-12 18:12:50.000000000 -0400
++++ linux-4.0/kernel/reboot.c	2015-05-06 09:45:56.991894271 -0400
+@@ -309,7 +309,7 @@
+ 	 * halt when pm_power_off is not set do it the easy way.
+ 	 */
+ 	if ((cmd == LINUX_REBOOT_CMD_POWER_OFF) && !pm_power_off)
+-		cmd = LINUX_REBOOT_CMD_HALT;
++		cmd = LINUX_REBOOT_CMD_RESTART;
```

Comment:
From IRC:
```
jell:  krnowak: without this patch - kernel poweroff treats as halt (infinite loop) if there is no pm_power_off (pm - power monitoring AIR)
```
In this case, maybe this patch could also add some comments about why it was changed.